src/AiLibrary.flix

```diff
@@ -90,33 +86,24 @@ namespace Flixball/AiLibrary {
 
     ///
     /// Selects a move based on the shortest path from position p to a
-    /// coordinate where `goal` is true.
+    /// coordinate where `goal` is true. Considers players obstacles if
+    /// `passablePlayers` is `false`.
```

Comment:
This should probably be a predicate eventually in case we add other terrain, etc. But ok for now.